### PR TITLE
Add `isOnHypertensionTreatment` column to `MedicalHistory` table

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/3920/add-isonhypertensiontreatment-column-to-medicalhistory

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -393,6 +393,7 @@
   <string name="newmedicalhistory_no">No</string>
   <string name="newmedicalhistory_history">History</string>
   <string name="newmedicalhistory_diagnosis">Diagnosis (Required)</string>
+  <string name="medicalhistory_question_is_on_htn_treatment">BP medicines taken any time in the last month?</string>
```
Comment (vinaysshenoy) [inline app/src/main/res/values/strings.xml:396]: Can we avoid acronyms like "htn"? Let's use the full word, please?

**Review by vinaysshenoy — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (vinaysshenoy) [reply app/src/main/res/values/strings.xml:396]: Approving this for now since it's urgent. We can rename this in another PR.

**Review by vinaysshenoy — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (msasikanth) [reply app/src/main/res/values/strings.xml:396]: Yep 👍🏾

**Comment by vinaysshenoy** [comment]
@msasikanth Looks like instrumented tests are failing, Please take a look?

**Review by vinaysshenoy — APPROVED** [review]

**Pull request merged**